lbrynet/tests/unit/daemon/test_claims_comparator.py

```diff
@@ -0,0 +1,27 @@
+import json
+import unittest
+
+from lbrynet.daemon.claims_comparator import arrange_results
+
+
+class ClaimsComparatorTest(unittest.TestCase):
+    def setUp(self):
+        with open('claims_comparator_cases.json') as f:
+            document = json.load(f)
+        self.cases = document['cases']
```

Comment:
What is the rationale for storing the dictionaries in a separate json file instead of just having this directly in the code?